cmd/kube-proxy/app/server_others.go

```diff
@@ -191,7 +191,7 @@ func newProxyServer(
 			// Just because the feature gate is enabled doesn't mean the node
 			// actually supports dual-stack
 			if _, err := ipt[1].ChainExists(utiliptables.TableNAT, utiliptables.ChainPostrouting); err != nil {
-				klog.Warningf("No iptables support for IPv6: %v", err)
+				klog.ErrorS(err, "No iptables support for IPv6")
```

Comment:
Maybe Warningf ->InfoS
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md